scripts/upload-secrets/upload_secrets.rb

```diff
@@ -8,10 +8,18 @@
 require "yaml"
 require "tempfile"
 
-def get_secret(path)
+def get_secret(path, *default_out)
   out = `pass "#{path}"`
-  abort out unless $CHILD_STATUS.success?
-  out
+  if $CHILD_STATUS.success?
```

Comment:
I would use `return` early to avoid nested conditionals

```
out = `pass "#{path}"`

return out if $CHILD_STATUS.success?
return default.to_s unless default.nil?

abort out

Reply:
done and done : )